rbac/management/group/view.py

```diff
@@ -47,6 +47,9 @@
 VALID_EXCLUDE_VALUES = ['true', 'false']
 VALID_GROUP_ROLE_FILTERS = ['role_name', 'role_description']
 VALID_GROUP_PRINCIPAL_FILTERS = ['principal_username']
+ORDERING_PARAM = 'order_by'
+VALID_ROLE_ORDER_FIELDS = ['name', 'modified', 'policyCount']
```

Comment:
I'm getting an error when I try to order by `policyCount`, it looks like because it's not actually a field on the role.

Reply:
Oh, that's odd. I pulled those from the fields in the model, I thought. I'll look back at it. Thanks!

Reply:
Oh, I encounter similar problem before. Our models does not have policyCount, it is annotated in the queryset . Adding annotation to the queryset would solve the problem.

Reply:
Thanks! Just got that added in